packages/11ty/_includes/components/navigation.js

```diff
@@ -173,8 +173,8 @@ module.exports = function(eleventyConfig) {
           </div>
         </nav>
         <div class="quire-progress-bar">
-          <div style="width: calc(100% * (${currentPageIndex + 1} / ${pages.length}));">
-            <span>${currentPageIndex + 1}/${pages.length}</span>
+          <div style="width: calc(100% * (${currentPageIndex + 1} / ${collections.navigation.length}));">
+            <span>${currentPageIndex + 1}/${collections.navigation.length}</span>
           </div>
```

Comment:
Does it makes sense elsewhere to set 
```
const pageNumber = currentPageIndex + 1
const pageCount = collections.navigation.length
const pagePercent = pageNumber/pageCount * 100
```
and here 
```suggestion
          <div style="width: pagePercent">
            <span>${pageNumber}/${pageCount}</span>
          </div>
```